Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -392,11 +405,55 @@ shortDistanceConfig state =
           { text = (getString YOU_CAN_TAKE_A_WALK_OR_CONTINUE_WITH_RIDE_BOOKING)
           , margin = (Margin 0 16 0 20)
           }
-        , option1 { text = (getString GO_BACK_) }
-        , option2 { text = (getString BOOK_RIDE_) }
+        , option1 
+        { text = (getString GO_BACK_) 
+        , fontSize = FontSize.a_16
+        , color = Color.black700
+        , strokeColor = Color.black700 
+        } 
+        , option2 
+        { text = (getString BOOK_RIDE_)
+        , fontSize = FontSize.a_16
+        , margin = (Margin 12 0 0 0)
+        }
+        }
+    popUpConfig2' =
+     config'
+        { gravity = CENTER
```

Comment:
Most the things in both configs are same. Please reuse common things. Instead of just creating the whole new view.